src/analyz/mod.rs

```diff
@@ -83,6 +82,27 @@ impl<'tcx> ToJson<'tcx> for mir::AggregateKind<'tcx> {
     }
 }
 
+pub fn merge(mut a: serde_json::Value, b: serde_json::Value) -> serde_json::Value {
```

Comment:
It might be worth documenting the preconditions that (1) both `Value`s must be JSON objects, and (2) the keys in each object must be disjoint.